### PR TITLE
Update README and sample apps with latest APIs

### PR DESCRIPTION
**Pull request by jhankin**

### CODE REVIEW

File: SquareRegisterSDK Test App/SCCTestAppDelegate.m [hunk 1]
```diff
@@ -66,21 +68,18 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
...
+                                         returnAutomaticallyAfterPayment:YES
                                                                    error:&error];
 
     // Perform the request.
-    BOOL const success = [SCCAPIConnection performRequest:request error:&error];
-    if (!success) {
-        UIAlertView *alertView = [[UIAlertView alloc] initWithTitle:@"Register app installed?" message:@"Make sure the register app is installed in the simulator. Log in as the merchant with token 7074ME2C077ZB." delegate:self cancelButtonTitle:@"Dismiss" otherButtonTitles:nil];
-        [alertView show];
-    }
+    [SCCAPIConnection performRequest:request error:&error];
```
Comment (amberdixon) [inline SquareRegisterSDK Test App/SCCTestAppDelegate.m]: You may want to continue to show the UIAlertView if an error occurs.

Comment (jhankin) [inline SquareRegisterSDK Test App/SCCTestAppDelegate.m]: Discussed offline; these errors are spuriously generated due to calling `performRequest:error` from `application:didFinishLaunchingWithOptions:`.  Will do a more holistic fix at a later date.

**Comment by amberdixon** [comment]
LGTM. There are some failing builds.

**Comment by jhankin** [comment]
I think that's due to a Travis CI configuration mismatch since I bumped the deployment target to 9.0.  Investigating.

**Comment by EricMuller22** [comment]
lgtm too

**Pull request merged**